test/ejabberd_tests/tests/rest_SUITE.erl

```diff
@@ -253,10 +233,73 @@ password_can_be_changed(Config) ->
     {?OK, _} = putt("/users/localhost/bob", #{newpass => <<"makrolika">>}),
     % now he logs again with the regular one
     escalus:story(Config, [{bob, 1}], fun(_Bob) ->
-            just_dont_do_anything
-        end),
+        just_dont_do_anything
+    end),
     ok.
 
+%%--------------------------------------------------------------------
+%% Helpers
+%%--------------------------------------------------------------------
+
+fill_archive(A, B) ->
+    % here we generate messages sent one, two and three days ago at 10am
+    {TodayDate, _} = calendar:local_time(),
+    Today = calendar:date_to_gregorian_days(TodayDate),
+    put_msg(A, B, <<"A">>, Today - 3),
+    put_msg(B, A, <<"A">>, Today - 3),
+    put_msg(A, B, <<"B">>, Today - 2),
+    put_msg(B, A, <<"B">>, Today - 2),
+    put_msg(A, B, <<"C">>, Today - 1),
+    put_msg(B, A, <<"C">>, Today - 1).
+
+put_msg(Aclient, Bclient, Content, Days) ->
+    DateTime = {calendar:gregorian_days_to_date(Days), {10, 0, 0}},
+    AArcId = make_arc_id(Aclient),
+    BArcId = make_arc_id(Bclient),
+    Msg = mam_helper:generate_msg_for_date_user(AArcId, BArcId, DateTime, Content),
+    put_msg(Msg),
+    ok.
+
+put_msg({{MsgIdOwner, MsgIdRemote},
+    {_FromBin, FromJID, FromArcID},
+    {_ToBin, ToJID, ToArcID},
+    {_, Source, _}, Packet}) ->
+    Host = escalus_ct:get_config(ejabberd_domain),
```

Comment:
Such domain reading should be slowly replaced by `ct:get_config({hosts, mim, domain})`.
You can also use the `host/0` function from this module which does exactly that.